### PR TITLE
Merged docs/commit-message.md into contribution-guidelines.md

### PR DESCRIPTION
**Pull request by dylanPowers**
commit-message.md duplicated a section in contribution-guidelines.md. I went ahead and merged the two. 

Merge #29 before this

### CODE REVIEW

File: contribution-guidelines.md [hunk 1]
```diff
@@ -69,7 +97,6 @@ License: MIT
 Signed-off-by: Juan Benet <juan@ipfs.io>
```
Comment (jbenet) [inline contribution-guidelines.md]: maybe add the

```
License: MIT
```
here too

**Comment by jbenet** [comment]
LGTM otherwise

File: contribution-guidelines.md (hunk 1 above)
Comment (dylanPowers) [inline contribution-guidelines.md]: The line above says "License: MIT". I'm confused?

Comment (jbenet) [inline contribution-guidelines.md]: oh sorry, the diff was hiding it. nvm

**Comment by chriscool** [comment]
Yeah, but the GitCop footer was pointing to commit-message.md, and now that this file is gone, the link gives a 404 not found.
I will fix that.

**Comment by jbenet** [comment]
Oh I'm sorry. that's my mistake. Apologies-- hard to keep track of all the dependencies on things.

@chriscool i'm ok to revert this, too. The "single file to look at" for figuring out what's wrong with the commit was pretty useful, i think. maybe we can just link to it from the original `contribution-guidelines.md`.

**Comment by chriscool** [comment]
@jbenet no problem I already fixed the GitCop footer to point to:

https://github.com/ipfs/community/blob/master/contribution-guidelines.md#commit-messages

But note that there is still some duplication as the commit-messages section now talks about the Signed-off-by: as does the sign-and-license-your-work section in the same document:

https://github.com/ipfs/community/blob/master/contribution-guidelines.md#sign-and-license-your-work

I will let you decide what is best.

**Comment by dylanPowers** [comment]
@chriscool Darn, I missed that. I'll quickly submit another pr

**Pull request merged**